xlml/apis/task.py

```diff
@@ -637,18 +637,18 @@ class GpuGkeTask(BaseTask):
   """This is a class to set up tasks for GPU on a GKE cluster.
 
   Attributes:
-    image_project: the project that an image belongs to.
-    image_family: the family group that an image belongs to.
+    task_test_config: task configutation.
+    task_gcp_config: gcp related config (e.g., zone, project) for the task.
     cluster_name: Name of the GCP cluster.
     job_create_timeout: Amount of time to wait for all pods to become active.
+    task_metric_config: metric configuration (e.g., result gcs path).
   """
 
-  task_test_config: test_config.JSonnetGpuTest
+  task_test_config: test_config.PlainGpuTest
```

Comment:
I hope you didn't break support for `JSonnetGpuTest`! If `task_test_config` can be either, then this should be a `Union`. Combining the two test types somehow (likely with different constructors) is also fine.

Reply:
`JSonnetGpuTest` is derived from `PlainGpuTest`:
```
@attrs.define(kw_only=True)
class JSonnetGpuTest(PlainGpuTest)
...
```
It's OK here I think?

Reply:
Oooh, I totally missed the inheritance relationship between them. What's the difference between them? Just the static constructor (`from_pytorch`)? In that case, can you just merge these two test configs into one type called `GpuGkeTest`?

Reply:
Merged into one.